src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article className={todo.completed
+    ? 'TodoInfo TodoInfo--completed'
+    : 'TodoInfo'}
```

Comment:
Don't repeat `TodoInfo` twice. You could use the classnames libraries. Or this option: 
```suggestion
  <article 
    className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
```